homeassistant/components/fronius/sensor.py

```diff
@@ -609,6 +611,7 @@ def __init__(
         """Set up an individual Fronius meter sensor."""
         super().__init__(coordinator)
         self.entity_description = self.entity_descriptions[key]
+        self.entity_id = f"{SENSOR_DOMAIN}.{DOMAIN}_{self._entity_id_prefix}_{self.entity_description.name}"
```

Comment:
We recommend not setting an entity id explicitly when setting unique id since the user can change both name and entity id easily via the frontend.

Regardless, this is not the PR to change this.